drivers/crypto/crypto_nxp_s32_hse.c

```diff
@@ -0,0 +1,660 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_s32_crypto_hse_mu
+
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/crypto/crypto.h>
+#include <Hse_Ip.h>
+
+#define LOG_LEVEL CONFIG_CRYPTO_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(crypto_nxp_s32_hse);
+
+#define CRYPTO_NXP_S32_HSE_SERVICE_TIMEOUT 1000000000
```

Comment:
```suggestion
#define CRYPTO_NXP_S32_HSE_SERVICE_TIMEOUT_MS 1000000000
```
to be consistent with the macro in line below (if it's not ms, use the corresponding time unit suffix)

Reply:
its unit is tick, I updated.